packages/xo-web/src/common/xo/index.js

```diff
@@ -1245,7 +1248,32 @@ export const migrateVm = (vm, host) =>
         _('migrateVmNoTargetHostMessage')
       )
     }
-    return _call('vm.migrate', { vm: vm.id, ...params })
+
+    return _call('vm.migrate', { vm: vm.id, ...params }).catch(err =>
+      confirm({
+        body: (
+          <div>
+            <p>{_('forceVmMigrateModalMessage')}</p>
+            <p>
+              <Tooltip content={_('showLogs')}>
+                <Link
+                  className='text-danger'
+                  target='_blank'
+                  to='/settings/logs'
+                >
+                  <Icon icon='alarm' /> {err.message || _('showLogs')}
```

Comment:
So this will always be `unknown error from the peer`? We should make sure to catch the correct error (different CPU) and show a (static) message about it.